GAME/TOMB4FX.C

```diff
@@ -204,6 +205,39 @@ int GetFreeDrip()
 
 }
 
+void TriggerLaraDrips()
+{
+	if (!(wibble & 0xF))
+	{
+		for(int i = 14; i >= 0; i--)
```

Comment:
All variables must be declared at the start of a method.

Reply:
C89 states that variables must be declared at the start of a scope block, which is the case here